compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/syntax/tree/SyntaxKind.java

```diff
@@ -201,23 +201,43 @@
     PARAMETER_DOC_REFERENCE_TOKEN(907, "parameter"),
     CONST_DOC_REFERENCE_TOKEN(908, "const"),
 
+    //Literal tokens
```

Comment:
Usually we keep a space between `//` and the first character in the comment :)

Reply:
Thanx will change.